python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1627,6 +1631,31 @@ def test_parquet_partitioned(tmpdir_factory, cols, filename):
                 assert fn == filename
 
 
+def test_parquet_writer_chunked_partitioned(tmpdir_factory):
```

Comment:
Should we add a test for the `ParquetDatasetWriter.close(return_metadata=True)` case?